examples/s3-import/locals.tf

```diff
@@ -4,11 +4,76 @@ locals {
   vpc_id                    = data.aws_vpc.supporting.id
   database_subnets          = local.database_subnet_id
   supporting_resources_name = "terraform-aws-rds"
+  partition                 = data.aws_partition.current.partition
+  dns_suffix                = data.aws_partition.current.dns_suffix
+  replication_bucket        = "${local.name}-replication-bucket"
+  aws_region_1              = "eu-west-1"
```

Comment:
Can we use a variable instead or is this string required here for the provider/data source blocks?

Reply:
we use a variable here for `aws region`